src/thunderbots/software/ai/hl/stp/stp_hl.cpp

```diff
@@ -1,13 +1,46 @@
 #include "ai/hl/stp/stp_hl.h"
+#include "ai/hl/stp/play/play.h"
+#include "ai/hl/stp/tactic/tactic.h"
 #include "ai/intent/move_intent.h"
 
 STP_HL::STP_HL()
 {
 }
 
-std::vector<std::unique_ptr<Intent>> STP_HL::getIntentAssignment(const World &world) const
+std::vector<std::pair<Robot, std::unique_ptr<Tactic>>> STP_HL::assignTacticsToRobots(
+    const World &world, const std::vector<std::unique_ptr<Tactic>> &tactics) const
 {
-    // TODO: Implement this
-    // https://github.com/UBC-Thunderbots/Software/issues/24
-    return std::vector<std::unique_ptr<Intent>>();
+    return std::vector<std::pair<Robot, std::unique_ptr<Tactic>>>();
+}
+
+std::shared_ptr<Play> STP_HL::calculateNewPlay(const World &world) const
+{
+    return Play::getRegistry().at(0)->getInstance();
+}
+
+std::vector<std::unique_ptr<Intent>> STP_HL::getIntentAssignment(const World &world)
+{
+    std::cout << "getting stp intent assignment" << std::endl;
```

Comment:
These couts should be removed; appears that they are for debugging.
Alternatively (to be tackled as a separate issue), we can add a configuration for debug statements, and #ifdef these out. Could be very useful in the future :P

Reply:
I believe we can do that with `ROS_DEBUG`. Will look in more detail later. @garethellis0 for his expertise.

Reply:
Could also be something like a environment variable that CMake can read, and then it can define something like `TBOTS_DEBUG`.

Reply:
In general, I don't think it's a good idea for us to role our own logging solution here. There are lots of well implemented libraries out there that we can use that are well documented and feature-rich (not to mention the fact that we don't have to maintain them). Some examples: https://github.com/google/glog, https://github.com/gabime/spdlog, https://github.com/KjellKod/g3log (a faster version of glog),

Reply:
Agree with the above. I've removed the print statement and created a new ticket for adding logging capabilities in #33